bindings_test.go

```diff
@@ -43,7 +43,7 @@ func TestBindingFloat64(t *testing.T) {
 		var out float64
 		var rows *RowsExtended
 		for _, v := range types {
-			dbt.mustExec(fmt.Sprintf("CREATE TABLE test (id int, value %v)", v))
+			dbt.mustExec(fmt.Sprintf("CREATE OR REPLACE TABLE test (id int, value %v)", v))
```

Comment:
If table exist will some rows inserted already, will it break the test, which expect only certain rows in the table?

Reply:
No :) `CREATE OR REPLACE` is very similar to `DROP` followed by `CREATE`. The only difference is the time travel feature, which is irrelevant in our tests. See red box [in docs](https://docs.snowflake.com/en/sql-reference/sql/create-table#usage-notes)